### PR TITLE
Replace shield icons for Mexican federal highways

### PR DESCRIPTION
**Pull request by claysmalley**
This PR replaces the shield icons for federal highways of Mexico. Since this shield is portrait-oriented and may contain up to 7 characters, I took the same approach from #675 and made the 2-digit variant 16 pixels wide, with the 3- and 4-digit variants stretched to 22 and 28 pixels, respectively.

---

Este PR reemplaza los íconos de escudo de las carreteras federales de México. Dado que este escudo está orientado verticalmente y puede contener hasta 7 caracteres, tomé el mismo enfoque del #675 e hice que la variante de 2 dígitos tuviera 16 píxeles de ancho, con las variantes de 3 y 4 dígitos ampliadas a 22 y 28 píxeles respectivamente.

### CODE REVIEW

**Comment by ZeLonewolf** [comment]
Screen shot before and after:

image

image

File: test/spec/shield.js [hunk 1]
```diff
@@ -27,8 +27,8 @@ const handler = shieldRenderer.getStyleImageMissingHandler();
 
 handler({ id: "shield\nBAB\n5\n\n" });
 handler({ id: "shield\nUS:RI\n\n\n" });
-handler({ id: "shield\nUS:RI\nABC123\n\n" });
-handler({ id: "shield\nUS:RI\nEquator\n\n" });
+handler({ id: "shield\nUS:RI\nABC1234\n\n" });
+handler({ id: "shield\nUS:RI\nLatitude\n\n" });
```
Comment (ZeLonewolf) [inline test/spec/shield.js:31]: Is this change intended?

File: test/spec/shield.js [hunk 2]
```diff
@@ -51,9 +51,9 @@ describe("shield", function () {
       expect(isBlankSprite("shield\nUS:RI\n\n\n")).to.be.true;
     });
     it("rejects a long ref", function () {
-      expect(mockRepo.hasSprite("shield\nUS:RI\nABC123\n\n")).to.be.true;
-      expect(isBlankSprite("shield\nUS:RI\nABC123\n\n")).to.be.false;
-      expect(isBlankSprite("shield\nUS:RI\nEquator\n\n")).to.be.true;
+      expect(mockRepo.hasSprite("shield\nUS:RI\nABC1234\n\n")).to.be.true;
+      expect(isBlankSprite("shield\nUS:RI\nABC1234\n\n")).to.be.false;
+      expect(isBlankSprite("shield\nUS:RI\nLatitude\n\n")).to.be.true;
```
Comment (ZeLonewolf) [inline test/spec/shield.js:56]: Is this changed intended?

File: test/spec/shield.js (hunk 1 above)
Comment (claysmalley) [reply test/spec/shield.js:31]: Yes; this is related to changing the max ref length to 7.

File: test/spec/shield.js (hunk 2 above)
Comment (claysmalley) [reply test/spec/shield.js:56]: Related to changing max ref length to 7.

**Review by ZeLonewolf — APPROVED** [review]
Looks good, thanks for working on this!
